com.unity.render-pipelines.core/Runtime/RenderGraph/RenderGraphDefaultResources.cs

```diff
@@ -32,6 +32,9 @@ public class RenderGraphDefaultResources
         /// <summary>Default white XR 2D texture.</summary>
         public TextureHandle whiteTextureXR { get; private set; }
 
+        /// <summary>Default opaue black XR 2D texture.</summary>
+        public TextureHandle opaqueBlackTextureXR { get; private set; }
```

Comment:
coud you comment why you need to create a "opaqueBlackTextureXR" ?

Reply:
I have added a comment in the code so it will be more clear to people working with the codebase.